appveyor.yml

```diff
@@ -26,13 +26,25 @@ cache:
 environment:
   global:
     TRIAL_REPORTER: text
+    TWISTED_REACTOR: default
```

Comment:
Not needed.  If **TWISTED_REACTOR** is not in the environment, tox.ini uses **default**